README.md

```diff
@@ -1 +1,49 @@
-This is the Videomap capstone project
+# Videomap
```

Comment:
Maybe include the deployed website link somewhere (http://videomap-step-2020.appspot.com/)

Reply:
second this, could be in parenthesis by # Videomap (i.e # Videomap (link))